backend/src/src-schedule/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) Bosch Software Innovations GmbH 2016.
+  ~ Part of the SW360 Portal Project.
+  ~
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ which accompanies this distribution, and is available at
+  ~ http://www.eclipse.org/legal/epl-v10.html
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <dependencies>
+        <dependency>
+            <groupId>com.siemens.sw360</groupId>
+            <artifactId>datahandler</artifactId>
+            <version>1.4.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>com.liferay.portal</groupId>
+            <artifactId>portal-service</artifactId>
+            <version>${liferay.version}</version>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
+
+    <parent>
+        <groupId>com.siemens.sw360</groupId>
+        <artifactId>backend-src</artifactId>
+        <version>1.4.0-SNAPSHOT</version>
+    </parent>
+
+    <developers>
+        <developer>
+            <name>birgit.heydenreich@tngtech.com</name>
+        </developer>
+        <developer>
+            <name>maximilian.huber@tngtech.com</name>
+        </developer>
```

Comment:
I think it would be fair to add Stefan to the list as he's mentioned as an author of several classes contained in this module.